### PR TITLE
fix: theme icon on first render

### PR DESCRIPTION
**Pull request by quentinderoubaix**

### CODE REVIEW

File: demo/src/routes/menu/Theme.svelte [hunk 1]
```diff
@@ -79,7 +80,9 @@
 			data-bs-toggle="dropdown"
 			aria-expanded={$open$}
 		>
-			<Svg svg={$currentTheme$.icon} />
```
Comment (andreascorti) [inline demo/src/routes/menu/Theme.svelte:82]: Maybe you can remove `icon` from the `currentTheme$` then, as it is not used anymore

**Review by andreascorti — APPROVED** [review]
LGTM @quentinderoubaix, left a very small comment, but you can merge without it

**Comment by quentinderoubaix** [comment]
Thanks for the review @andreascorti !

File: demo/src/routes/menu/Theme.svelte (hunk 1 above)
Comment (quentinderoubaix) [reply demo/src/routes/menu/Theme.svelte:82]: Indeed, maybe the currentTheme$ should just be a ```writable(string)```,  and would be the id of the current theme.

**Pull request merged**